airflow/providers/google/cloud/operators/cloud_build.py

```diff
@@ -83,13 +85,29 @@ def _reformat_storage_source(self):
 
         self.body["source"]["storageSource"] = self._convert_storage_url_to_dict(source)
 
+    def _load_body_to_dict(self):
+        """
+        :param file:
+            file path to YAML build config
+        :return: dict
+        """
+        try:
+            with open(self.body, 'r') as f:
```

Comment:
I think, we should use [template_ext](https://airflow.readthedocs.io/en/latest/_api/airflow/models/baseoperator/index.html?highlight=BaseOperator#airflow.models.baseoperator.BaseOperator.template_ext) to load file. This will cause us to still have text instead of the dictionary, so we need to overwrite the [prepare_template method](https://airflow.readthedocs.io/en/latest/_api/airflow/models/baseoperator/index.html?highlight=BaseOperator#airflow.models.baseoperator.BaseOperator.prepare_template) to convert the string to a dictionary